pkg/security/ebpf/probes/event_types.go

```diff
@@ -149,10 +149,12 @@ func GetSelectorsPerEventType(fentry bool) map[eval.EventType][]manager.ProbesSe
 			// Open probes
 			&manager.AllOf{Selectors: []manager.ProbesSelector{
```

Comment:
Shouldn't we use `OneOf` instead of `AllOf` here?

Reply:
Not yet :( I first need to find a stable hook point or a combination of hook points that will cover the ftruncate syscall.

Reply:
Well idk if you intuited the answer or if you looked it up, but you ended up being almost right 😅 
For some reason, they introduced `security_file_truncate` in 6.2, and replaced `security_path_truncate` with it only for `do_sys_ftruncate`. So in the end, changing it to "besteffort" should do the trick ! ("oneof" would prevent hooking both of them when they're available - which we need to support ftruncate on recent kernels)

Reply:
Are you sure OneOf prevents the hooking of both ? My understanding was that it was kind of "at least one" semantics